pygmt/helpers/utils.py

```diff
@@ -15,127 +15,133 @@
 from pygmt.exceptions import GMTInvalidInput
 
 
-def _validate_data_input(
-    data=None, x=None, y=None, z=None, required_z=False, required_data=True, kind=None
+def validate_data_input(
```

Comment:
I think it's more useful to pass the list of column names instead, i.e., replacing `ncols=2` with `names=["x", "y"]`.

So, for most modules, `vectors=["x", "y"]` and `names=["x", "y"]` or `vectors=[x, y, z]` and `names=["x", "y", "z"]`.

For more complicated modules like `plot` or `plot3d`, the names can be 
`names=["x", "y", "direction_arg1", "direction_arg2", "fill", "size", "symbol", "transparency"]`.

The column names will be very useful when the GMTInvalidInput exception is raised.
For example, instead of `"Column 5 can't be None."`, we can say `"Column 5 ('size') can't be None."`. Instead of `"data must have at least 8 columns."`, we can say
```
data must have at least 8 columns:
x y direction_arg1 direction_arg2 fill size symbol transparency
```

Reply:
Done in f37413b2aa03c24de6c9ea1e94053d7607b4d9b2